crates/aptos-faucet/src/main.rs

```diff
@@ -211,9 +211,13 @@ mod tests {
         accounts: AccountStates,
     ) -> Result<impl Reply, Rejection> {
         let reader = accounts.read();
-        let account = AccountAddress::try_from(address)
-            .ok()
-            .and_then(|address| reader.get(&address));
+        let account = match AccountAddress::try_from(address.clone()) {
+            Ok(addr) => reader.get(&addr),
+            _ => match AccountAddress::from_hex(address.clone()) {
+                Ok(addr) => reader.get(&addr),
+                _ => None,
+            },
+        };
```

Comment:
use or_else. There are other places using the similar way doing conversion. https://github.com/aptos-labs/aptos-core/issues/534 is trying to solve the problem so that we don't need to try two conversion at the same time. will postphone the refactoring in 534